runtime/onert/core/src/exec/ExecutionObservers.cc

```diff
@@ -74,7 +74,17 @@ ChromeTracingObserver::ChromeTracingObserver(const std::string &filepath, const
 {
 }
 
-ChromeTracingObserver::~ChromeTracingObserver() { _recorder.writeToFile(_ofs); }
+ChromeTracingObserver::~ChromeTracingObserver()
+{
+  try
+  {
+    _recorder.writeToFile(_ofs);
+  }
+  catch (const std::runtime_error &e)
```

Comment:
`Json::LogicError` is inherited from `std::exception`, not `std::runtime_error`. Please change it to `std::exception`
Refer to declaration as below
`JSONCPP_NORETURN void throwLogicError(JSONCPP_STRING const &msg) { throw LogicError(msg); }`
`class JSON_API LogicError : public Exception`
`class JSON_API Exception : public std::exception`